app/models/state_file_dependent.rb

```diff
@@ -32,20 +32,33 @@
 #
 
 class StateFileDependent < ApplicationRecord
-  # TODO: once we have added all the json fixtures for all the states we can remove RELATIONSHIP_LABELS used to map xml relationship to gender-neutral terms in relationship colum.
-  #
+
   RELATIONSHIP_LABELS = {
-    "DAUGHTER" => "Child",
-    "STEPCHILD" => "Child",
-    "FOSTER CHILD" => "Foster Child",
-    "GRANDCHILD" => "Grandchild",
-    "SISTER" => "Sibling",
-    "HALF SISTER" => "Half-Sibling",
-    "NEPHEW" => "Niece/Nephew",
-    "STEPBROTHER" => "Step-Sibling",
-    "PARENT" => "Parent",
-    "GRANDPARENT" => "Grandparent",
-    "NONE" => "Other",
+    "biologicalChild" => "Child",
+    "adoptedChild" => "Child",
+    "stepChild" => "Child",
+    "fosterChild" => "Foster Child",
+    "grandChildOrOtherDescendentOfChild" => "Grandchild",
+    "childInLaw" => "Child",
+    "sibling" => "Sibling",
+    "childOfSibling" => "Niece/Nephew",
+    "halfSibling" => "Half-Sibling",
+    "childOfHalfSibling" => "Niece/Nephew",
+    "stepSibling" => "Step-Sibling",
+    "childOfStepSibling" => "Niece/Nephew",
+    "otherDescendantOfSibling" => "Niece/Nephew",
+    "siblingInLaw" => "Sibling",
+    "parent" => "Parent",
+    "grandParent" => "Grandparent",
+    "otherAncestorOfParent" => "Grandparent",
+    "stepParent" => "Parent",
+    "parentInLaw" => "Parent",
+    "noneOfTheAbove" => "Other",
+    "siblingOfParent" => "?",
+    "otherDescendantOfHalfSibling" => "?",
+    "otherDescendantOfStepSibling" => "?",
+    "fosterParent" => "?",
+    "siblingsSpouse" => "?",
```

Comment:
how is the `?` value used?

Reply:
Good catch - I meant to replace these with something and didn't get back to that before asking for review. I'll map these to what I think are sensible values and ask for PM input on the story.